package_manifest.yaml

```diff
@@ -0,0 +1,391 @@
+---
+packages:
+  children:
+    foreman: {}
```

Comment:
so uh, a group foreman, and a host foreman, that is gona explode. same for katello. :(

(that's why the pulp group is called pulp_group in sat-packaging)